#!/usr/bin/env python
PACKAGE = "obstacle_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("isShow_", bool_t, 0, "Visual segments", False)
gen.add("scanRange_", double_t, 0, "Limit scan range", 2, 0.3, 5)
gen.add("sacThreshold_", double_t, 0, "Distance to the model threshold", 0.01, 0.01, 0.05)
gen.add("minObjHeight_", double_t, 0, "Distance will be deleted from the floow", 0.03, 0.01, 1.0)
gen.add("tolerance_", double_t, 0, "KDTree tolerance", 0.02, 0.005, 0.1)

exit(gen.generate(PACKAGE, "PclDetectorNode", "PclDetector"))
